cli/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 
 _Released 4/2/2024 (PENDING)_
 
+**Performance:**
```

Comment:
Is this really performance? Or more stability?

Reply:
A little of A, a little of B - but the source issue is labeled performance